src/ontology/doid-edit.owl

```diff
@@ -40307,6 +40317,73 @@ AnnotationAssertion(rdfs:label obo:DOID_0060911 "karyomegalic interstitial nephr
+AnnotationAssertion(rdfs:label obo:DOID_0060916 "proteasome-associated autoinflammatory syndrome 3"@en)
+SubClassOf(obo:DOID_0060916 obo:DOID_0050737)
+SubClassOf(obo:DOID_0060916 obo:DOID_0060919)
+SubClassOf(obo:DOID_0060916 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060917 (facioscapulohumeral muscular dystrophy 3)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060917 "A facioscapulohumeral muscular dystrophy characterized by adult onset of proximal muscle weakness affecting the face, neck, scapular muscles, and upper and lower limbs that has_material_basis_in the combination of a homozygous mutation in the LRIF1 gene on chromosome 1p13 and presence of a haplotype on chromosome 4 that is permissive for DUX4 expression.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060917 "OMIM:619477")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060917 "https://pubmed.ncbi.nlm.nih.gov/32467133/")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060917 "FSHD3")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060917 "facioscapulohumeral muscular dystrophy type 3")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060917 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060917 "DOID:0060917")
+AnnotationAssertion(rdfs:label obo:DOID_0060917 "facioscapulohumeral muscular dystrophy 3"@en)
+SubClassOf(obo:DOID_0060917 obo:DOID_11727)
+SubClassOf(obo:DOID_0060917 ObjectSomeValuesFrom(obo:IDO_0000664 obo:GENO_0000930))
+
+# Class: obo:DOID_0060918 (facioscapulohumeral muscular dystrophy 4)
+
+AnnotationAssertion(obo:IAO_0000115 obo:DOID_0060918 "A facioscapulohumeral muscular dystrophy characterized by adult onset of progressive muscle weakness of the face and upper extremity muscles with disease progression that has_material_basis_in the combination of a heterozygous mutation in the DNMT3B gene on chromosome 20q11 and presence of a haplotype on chromosome 4 that is permissive for DUX4 expression.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060918 "OMIM:619478")
+AnnotationAssertion(oboInOwl:hasDbXref obo:DOID_0060918 "https://pubmed.ncbi.nlm.nih.gov/27153398/")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060918 "FSHD4")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:DOID_0060918 "facioscapulohumeral muscular dystrophy type 4")
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060918 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060918 "DOID:0060918")
+AnnotationAssertion(rdfs:label obo:DOID_0060918 "facioscapulohumeral muscular dystrophy 4"@en)
+SubClassOf(obo:DOID_0060918 obo:DOID_11727)
+
+# Class: obo:DOID_0060919 (autoinflammatory disease)
+
+AnnotationAssertion(oboInOwl:hasOBONamespace obo:DOID_0060919 "disease_ontology")
+AnnotationAssertion(oboInOwl:id obo:DOID_0060919 "DOID:0060919")
+AnnotationAssertion(rdfs:label obo:DOID_0060919 "autoinflammatory disease"@en)
+SubClassOf(obo:DOID_0060919 obo:DOID_2914)
```

Comment:
Still not sure this disease should be added. Can you suggest 1-3 existing possible parents of the proteasome-associated autoinflammatory syndrome diseases being added?